src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+import java.util.List;
+
+public abstract class AbstractManager extends AbstractEmployee {
+
+    private final Float budget = 100_000f;
+    private List <AbstractEmployee> EmployeeList = getEmployeeList();
```

Comment:
In Java variable names should be written in camelCase, so:  `EmployeeList` -> `employeeList`